Documentation/planning/multi-sdk-band-support.md

```diff
@@ -0,0 +1,253 @@
+- The 1xx SDK is the only SDK that is supported for building the runtime components. Because tooling differs between the bands, using a newer band to build the runtime is likely to expose new warnings or issues.
+- There are components of the 1xx band that are used in the runtime. There are "sdk-like" components present in the runtime (analyzers, generators, etc.) that are dependent upon "sdk" functionality. Substituting in a 2xx aligned roslyn, for instance, would result in a product that differs from Microsoft's build.
+
+### What artifacts should be passed with --with-current-artifacts/--with-previous-artifacts?
+
+When building a full VMR (no skipped components), only --with-previous-artifacts should be passed. When building a subsetted VMR (e.g. 2xx, 3xx, etc.), both `--with-current-artifacts` and `--with-previous-artifacts` should be passed.
+
+`--with-current-artifacts` - The NuGet and non-NuGet archive outputs of the 1xx VMR build that is logically part of the same product iteration.
+`--with-previous-artifacts` - The NuGet outputs of the previous VMR build(s) that are logically part of the previous product iteration. This set may encompass more than one set of build outputs.
+
+### Previews and major release GA builds
+
+**No Change**
+
+A distro maintainer sees no change here. Until a few months **after** GA, only the 1xx band ships. The workflow remains the same as today. Clone the VMR branch/tag in question, build using previously source-built artifacts and SDKs.
+
+### A distro maintainer wishing to only support the 1xx SDK
+
+**No change**
+
+### A distro maintainer wishing to build the initial release (n00) of a Nxx+ band SDK
+
+The initial release of a Nxx+ band SDK should be straightforward. It is unlikely (though possible) that the initial Nxx band would depend on a Nxx SDK to build. If this is not the case, then a distro maintainer performs the following actions, and no bootstrap will be required:
+1. **Checkout** 1xx branch:
+2. **Build** using `./build.sh --with-sdk <PSB 1xx SDK> --with-previous-artifacts <PSB 1xx artifacts>`.
+3. **Gather** 1xx build outputs from `artifacts/x64/Release` = `<CSB 1xx artifacts>`.
+4. **Checkout** Nxx branch
+5. **Build** using `./build.sh --with-sdk <PSB 1xx SDK> --with-previous-artifacts <PSB 1xx artifacts> --with-current-artifacts <CSB 1xx artifacts>`
+
+If a Nxx SDK is required to build the initial Nxx SDK, then the following workflow is used:
+
+1. **Checkout** 1xx branch:
+2. **Build** using `./build.sh --with-sdk <PSB 1xx SDK> --with-previous-artifacts <PSB 1xx artifacts>`.
+3. **Gather** 1xx build outputs from `artifacts/x64/Release` = `<CSB 1xx artifacts>`.
+4. **Checkout** Nxx branch
+5. **Build** using `./build.sh --with-previous-artifacts <PSB 1xx artifacts> --with-current-artifacts <CSB 1xx artifacts>`
```

Comment:
maybe highlight this is with 3rd party SDK?

**Build** with **bootstrap SDK** using `./build.sh --with-previous-artifacts <PSB 1xx artifacts> --with-current-artifacts <CSB 1xx artifacts>`

Reply:
Done, thanks.